packages/docs/src/stories/templates/header.stories.ts

```diff
@@ -195,11 +209,25 @@ export const SampleA02 = {
         </sd-navigation-item>
       </nav>
     </sd-drawer>
-    <script>
-      function openDrawerSampleA2() {
-        const drawerSampleA2 = document.getElementById('sample-a02-drawer');
-        drawerSampleA2.show();
-      }
+    <script type="module">
+      await Promise.all([
+        customElements.whenDefined('sd-navigation-item'),
+        customElements.whenDefined('sd-drawer')
+      ]).then(() => {
+        const drawerSampleA02 = document.getElementById('sample-a02-drawer');
+        const navigationItemSampleA02 = document.getElementById('open-menu-sample-a-02');
+        const buttonSampleA02 = navigationItemSampleA02.shadowRoot.querySelector('button');
+
+        buttonSampleA02.setAttribute('aria-controls', 'sample-a02-drawer');
+        //Add the necessary ARIA attributes to prevent only being added after action
+        buttonSampleA02.setAttribute('aria-expanded', 'false');
+        drawerSampleA02.addEventListener('sd-hide', () => buttonSampleA02.setAttribute('aria-expanded', 'false'));
+
+        navigationItemSampleA02.addEventListener('click', () => {
+          drawerSampleA02.show();
+          buttonSampleA02.setAttribute('aria-expanded', 'true');
```

Comment:
Same here: navigation-item has to get aria attributes

Reply:
Same here https://github.com/solid-design-system/solid/pull/1668#discussion_r1863120614